bindings/haskell/README.md

```diff
@@ -0,0 +1,52 @@
+[![CI](https://github.com/donsbot/if-proto-scip-hs/actions/workflows/ci.yml/badge.svg)](https://github.com/donsbot/if-proto-scip-hs/actions/workflows/ci.yml)
+
+# Haskell bindings for SCIP
+
+These bindings use Google's proto-lens generator for Haskell. More details at:
+
+https://github.com/google/proto-lens#readme
+
+Building: see the following workflow file for most up to date:
+https://github.com/donsbot/if-proto-scip-hs/blob/main/.github/workflows/ci.yml
+
+First, get a working GHC environment:
+
+```sh
+# Linux
+GHCUPVERSION=0.1.17.4
+curl --proto '=https' --tlsv1.2 -sSf https://downloads.haskell.org/~ghcup/$GHCUPVERSION/x86_64-linux-ghcup-$GHCUPVERSION > /usr/bin/ghcup && \
+chmod +x /usr/bin/ghcup
+ghcup install ghc 8.10.7 --set
+ghcup install cabal 3.6.0.0 --set
+```
+
+```sh
+# macOS
+GHCUPVERSION=0.1.17.4
+curl --proto '=https' --tlsv1.2 -sSf https://downloads.haskell.org/~ghcup/$GHCUPVERSION/x86_64-apple-darwin-ghcup-$GHCUPVERSION > /usr/local/bin/ghcup && \
```

Comment:
suggestion: Maybe we should just recommend the official command?

```
curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
```

That is cross-platform (direct copy-pasting works for non x86_64 too), and will update for newer ghcup versions which are unlikely to break this basic functionality.

We already mention the CI script earlier, so one can look there for an exact version if desired.